[TECH] Bumper les paquets (PIX-2215).

sbedeau opened the pull request:
## :unicorn: Problème
Les paquets ne sont pas à jour.

## :robot: Solution
Bumper les paquets

## :rainbow: Remarques
RAS

## :100: Pour tester
Se balader

Commit:
chore: bump nuxt to latest

Commit:
chore: bump sass-loader to latest patch

Commit:
chore: bump nuxtjs/axios

Commit:
chore: bump fortawesome packages to latest

Commit:
chore: bump eslint packages

Commit:
chore: bump nuxt-i18n package to latest

Commit:
chore: bump nuxt-matomo to latest

Commit:
chore: bump eslint-config-prettier

Commit:
chore: bump lodash

Commit:
chore: bump autoprefixer

Commit:
chore: bump @vue/test-utils

Commit:
chore: bump postcss-cli

pix-service commented:
I'm deploying this PR to these urls:

- Pix Site (fr): https://site-pr242.review.pix.fr
- Pix Site (org): https://site-pr242.review.pix.org
- Pix Pro (fr): https://pro-pr242.review.pix.fr
- Pix Pro (org): https://pro-pr242.review.pix.org

 Please check it out!

alexandrecoin committed:
fix(eslintrc): follow new recommendation of eslint-config-prettier

https://github.com/prettier/eslint-config-prettier/blob/HEAD/CHANGELOG.md

alexandrecoin committed:
fix(tests): add async/await for setProps rendering

cf https://github.com/vuejs/vue-test-utils/pull/1752 that was added sneakily to 1.1.3

MelanieMEB reviewed (approved):
Testé sous IE : règle le soucis de clic sur le menu et sur la pop-in
